msgpack/fallback.py

```diff
@@ -302,7 +297,7 @@ def feed(self, next_bytes):
             self._buff_i -= self._buf_checkpoint
             self._buf_checkpoint = 0
 
-        self._buffer += view
+        self._buffer += bytearray(view)
```

Comment:
Doesn't this increase data copy?

Reply:
That is an interesting question.  My assumption is that the inplace add / concatenation operator is doing an implicit typecast for you in the background rather than throwing TypeError in newer python, so an explicit typecast was just replacing a step that had to happen implicitly somewhere anyway.  I don't actually want to read a bunch of code to verify that assumption though.  I figured out that .extend() works reliably with implicit typecasting in old and new which should be about an even trade.